### PR TITLE
Added images for rooms#7

### LINKED ISSUE #7: Rooms overview show room image
## User story

**Who:** **As a** user

**What:** **I want to** see the image of a room

**Why:** **so that we can** have a visual representation of the room

## Implementation details
- In mockup 1 beneath search, there is the overview of the rooms. The rooms overview will be implemented [here](https://github.com/HackYourFuture-CPH/chattie/issues/8)
- These room overviews should now support images
- Make sure to talk to the person responsible for the issue above. You will be working on top of that code, so make sure you understand how it works

### PR DESCRIPTION
**Pull request by Aswini-D3**
# Description
We can see the images of the rooms here...so that we can have a visual representation of the room.

Fix #7 
npm run storybook

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/CreateANewRoomButton/CreateANewRoomButton.js [hunk 1]
```diff
@@ -13,5 +13,4 @@ function CreateANewRoomButton() {
     </div>
   );
 }
-
```
Comment (tpetrina) [inline src/client/components/CreateANewRoomButton/CreateANewRoomButton.js:16]: Why did you remove this empty line?

File: src/client/components/RoomList/RoomList.css [hunk 2]
```diff
@@ -1,25 +1,23 @@
-button {
+img {
```
Comment (tpetrina) [inline src/client/components/RoomList/RoomList.css:1]: Do not style _all_ images in the app, have a better CSS selector only for this component.

**Review by bytewiz — CHANGES_REQUESTED** [review]

**Review by benna100 — APPROVED** [review]
You are still adding lots of small changes to the PR (line breaks, importing things other places, etc). Please avoid that for another time!

**Comment by Aswini-D3** [comment]
> You are still adding lots of small changes to the PR (line breaks, importing things other places, etc). Please avoid that for another time!

Thank you Benjamin..I wont repeat those next time.

**Pull request merged**